articles/custom_storage_location.md

```diff
@@ -263,7 +264,7 @@ fileHandle <- list(concreteType='org.sagebionetworks.repo.model.file.S3FileHandl
                    storageLocationId = destination$storageLocationId,
                    bucketName = destination$bucket,
                    key ='s3ObjectKey')
-fileHandle <- synRestPOST('/externalFileHandle/s3', body=toJSON(fileHandle), endpoint = synapseFileServiceEndpoint())
+fileHandle <- synRestPOST('/externalFileHandle/s3', body=toJSON(fileHandle), endpoint = 'https://file-prod.prod.sagebase.org/file/v1')
```

Comment:
synapseFileServiceEndpoint() doesn't work (at least for me) in synapser, so I removed references to it.

Reply:
why is this the endpoint and not the default endpoint?

Reply:
resolving to merge PR